lua/neo-tree/ui/popups.lua

```diff
@@ -102,13 +102,12 @@ M.alert = function(title, message, size)
 
   local success, msg = pcall(vim.api.nvim_buf_set_lines, win.bufnr, 0, 0, false, lines)
   if success then
-    win:map("n", "<esc>", function(bufnr)
-      win:unmount()
-    end, { noremap = true })
-
-    win:map("n", "<enter>", function(bufnr)
-      win:unmount()
-    end, { noremap = true })
+    local exit_keys = { "<esc>", "<enter>", "q" }
```

Comment:
This is reasonable. The only issue is that now the message on line 92 is out of sync with the mappings.